_posts/en/newsletters/2019-10-30-newsletter.md

```diff
@@ -71,7 +71,24 @@ answers posted since our last update.*
 endcomment %}
 {% assign bse = "https://bitcoin.stackexchange.com/a/" %}
 
-FIXME
+- [Why does hashing public keys not actually provide any quantum resistance?]({{bse}}91049)
+  Andrew Chow lists several considerations regarding public keys and quantum
+  resistance including: needing to reveal the public key during spending, the
+  large number of bitcoins in outputs with known public keys, and numerous ways
+  which public keys are exposed outside of just transacting due to not currently
+  being treated as secrets.
+
+- [Will schnorr multi-signatures completely replace ECDSA?]({{bse}}90855)
+  Ugam Kamat explains how the proposed addition of schnorr signatures in
+  segwit v1 does not remove the need for ECDSA. ECDSA is required to spend
+  non-segwit as well as segwit v0 outputs and its removal would also require a
+  hard fork.
```

Comment:
The SE answer does indeed say that removing ECDSA would require a hard fork, but it's wrong.  I'll argue over there, but I suggest changing it here.

Reply:
Agree. Harding has pointed out on SE that it might just be confusing wording, but I'd just remove entirely here. Perhaps change the sentence to "ECDSA will continue to be required to spend non-segwit as well as segwit v0 outputs."